### PR TITLE
Autoload.lua extensions improvement

### PR DESCRIPTION
**Pull request by thebombzen**
Alphebatize the extensions in autoload.lua's list of extensions and add several more.

### CODE REVIEW

File: TOOLS/lua/autoload.lua [hunk 1]
```diff
@@ -49,16 +49,18 @@ function SetUnion (a,b)
...
+    'ogm', 'opus', 'spx', 'wav', 'wma'
 }
 
 EXTENSIONS_IMAGES = Set {
-    'bmp', 'gif', 'jpeg', 'jpg', 'png', 'svg', 'tif', 'tiff', 'webp'
+    'avif', 'bmp', 'gif', 'j2k', 'jp2', 'jpeg', 'jpg', 'jxl', 'png',
+    'svg', 'tga', 'tif', 'tiff', 'webp'
 }
 
 EXTENSIONS = Set {}
```
Comment (sfan5) [inline TOOLS/lua/autoload.lua:66]: seems fine but two things:
I doubt anyone actually has any `spx` or `svg` files laying around

Comment (thebombzen) [reply TOOLS/lua/autoload.lua:66]: svg was already there, I can remove speex though

**Comment by Natural-Harmonia-Gropius** [comment]
what about add `'alac', 'dsd', 'dsf', 'dff'` to audio and `'jfif', 'jxr', 'heif', 'heic'` to image?
alac - apple music
dsd, dsf, dff - sony cd
jfif - twitter
jxr - hdr with xbox game bar screenshot
heif, heic - phone camera

**Comment by thebombzen** [comment]
> what about add `'alac', 'dsd', 'dsf', 'dff'` to audio and `'jfif', 'jxr', 'heif', 'heic'` to image? alac - apple music dsd, dsf, dff - sony cd jfif - twitter jxr - hdr with xbox game bar screenshot heif, heic - phone camera

ALAC is a codec, not a container, and its files typically have the `.m4a` extension. DSD, DSF, and DFF are extremely obscure.

JFIF is an extension that is not used (this is just an extremely rare extension used for jpeg images).

JPEG XR, HEIF, and HEIC are not supported by FFmpeg and mpv cannot even read them.

File: TOOLS/lua/autoload.lua (hunk 1 above)
Comment (sfan5) [reply TOOLS/lua/autoload.lua:66]: do whatever you deem correct :shrug:

**Pull request merged**